projects/go-lib/src/lib/components/go-icon/go-icon.component.spec.ts

```diff
@@ -22,4 +22,56 @@ describe('GoIconComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  describe('classObject()', () => {
+    it('builds an object with a modifier class key if iconModifier exists', () => {
+      expect(component.classObject()).toEqual({});
+
+      component.iconModifier = 'positive';
+
+      expect(component.classObject()).toEqual({
+        'go-icon--positive': true
+      });
+    });
+
+    it('builds an object with a additional classes key if iconClass exists', () => {
+      expect(component.classObject()).toEqual({});
+
+      component.iconClass = 'odd multiple class use case';
+
+      expect(component.classObject()).toEqual({
+        'odd multiple class use case': true
+      });
+    });
+  });
+
+  describe('the template', () => {
```

Comment:
Testing templates probably isn't something we need to do often, but it seemed so easy in this case that I figured we might as well.